src/app/shared/change-password-dialog/change-password-dialog.component.ts

```diff
@@ -0,0 +1,45 @@
+/** Angular Imports */
+import { Component, OnInit, Inject } from '@angular/core';
+import { MatDialogRef, MAT_DIALOG_DATA } from '@angular/material';
+import { FormBuilder, Validators } from '@angular/forms';
+
+/** Custom Imports */
+import { PasswordCheckService } from './password-check.service';
+
```

Comment:
please change the width and height to offer better ux
![Screenshot from 2020-05-13 20-24-05](https://user-images.githubusercontent.com/54709463/81830190-635eb300-9559-11ea-972c-218861209a43.png)

Reply:
Can you put the required width and the height?

Reply:
![Screenshot from 2020-05-13 21-44-56](https://user-images.githubusercontent.com/43112139/81837904-11228f80-9563-11ea-9ac9-ea1eff5d2029.png)

Is this okay?

Reply:
@karantakalkar Please review

Reply:
good